aiida/tools/groups/paths.py

```diff
@@ -257,7 +257,7 @@ def walk_nodes(
 
         :param filters: filters to apply to the node query
         :param node_class: return only nodes of a certain class (or list of classes)
-        :param int batch_size: The size of the batches to ask the backend to batch results in subcollections.
+        :param query_batch: The size of the batches to ask the backend to batch results in subcollections.
```

Comment:
`batch_size` might actually be the better name; anyhow it's fine to adapt the docs instead.

Reply:
yeh it probably should be, but thats outside the scope of this PR